include/z64.h

```diff
@@ -852,27 +849,51 @@ typedef struct {
 } PolygonDlist2; // size = 0x8
 
 typedef struct {
-    /* 0x00 */ u8    type;
+    /* 0x00 */ PolygonBase base;
     /* 0x01 */ u8    num; // number of dlist entries
     /* 0x04 */ void* start;
     /* 0x08 */ void* end;
 } PolygonType2; // size = 0xC
 
 typedef union {
-    Polygon      polygon;
+    PolygonBase  base;
     PolygonType0 polygon0;
     PolygonType1 polygon1;
     PolygonType2 polygon2;
-} Mesh; // "Ground Shape"
+} MeshHeader; // "Ground Shape"
+
+typedef enum {
+    /* 0 */ LENS_ACTORS_MODE_SEETHROUGH, // lens actors are visible by default and hidden by using lens (for example, fake walls)
+    /* 1 */ LENS_ACTORS_MODE_REVEAL // lens actors are invisible by default and revealed by using lens (for example, invisible enemies)
+} LensActorsMode;
```

Comment:
I cant explain exactly why it sounds wrong, but imo `lensActorsMode` -> `lensActorMode`
even though its true you are talking about multiple actors (all actors that react to lens in this case), the extra plural in the middle just sounds wrong. I guess actor in this context doesnt need to be plural to mean all of them, its just the name of the mode
"Im changing the lens actor mode" vs "Im changing the lens actors mode", the first is more natural